var/spack/repos/builtin/packages/process-in-process/package.py

```diff
@@ -35,47 +37,47 @@ class ProcessInProcess(Package):
     #  PiP-glibc resource
     #   for rhel/centos 7
     resource(name='PiP-glibc',
-             git='https://github.com/RIKEN-SysSoft/PiP-glibc.git',
+             git='git@github.com:procinproc/PiP-glibc.git',
```

Comment:
Can you leave it at https? Otherwise it requires ssh authentication.

Reply:
Oh, yes, you are right.  I will fix this.  

BTW, is it better to use hub?